libs/services/designer-client-services/src/lib/consumption/manifests/apiManagement.ts

```diff
@@ -66,9 +66,14 @@ export const apiManagementActionManifest = {
               'x-ms-dynamic-list': {
                 dynamicState: {
                   operationId: 'getApimOperations',
-                  parameters: {},
                 },
-                parameters: {},
+                parameters: {
+                  type: 'object',
+                  apiId: {
+                    parameterReference: 'api.id',
+                    required: true,
```

Comment:
Hoping these changes are tested in consumption with dynamic data.

Reply:
They were yes